server/datastore/mysql/mdm_test.go

```diff
@@ -2535,6 +2535,8 @@ func testBulkSetPendingMDMHostProfiles(t *testing.T, ds *Datastore) {
 		},
 	})
 
+	// TODO(mna): add exclude-any labels here, test the broken label behavior (Win+Mac)
```

Comment:
Just pointing out that the tests are not implemented yet, my plan is to add them next week but if we need to complete this earlier, Datastore tests would go here.